src/main/resources/static/data-mapping.js

```diff
@@ -132,6 +132,13 @@ $(document).ready(function() {
         }
     }
 
+    $('#map-proceed-btn').click(function() {
+        var isValid = true;
+        if(isValid) {
+            window.location.replace("/patients");
```

Comment:
I think this will not work on the server. This will change current url to host/patients, which is ocdu-dev/patients in case of our  server. You should add application name "ocdu" to it. Example
 
    var relPath = baseApp+"/patients";
 Preferably keep it in some central place so you need to change it only once when we need to change it.